lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -28,11 +28,18 @@
                     this.languages = options.teamParams.languages;
                     this.countries = options.teamParams.countries;
                     this.primaryButtonTitle = options.primaryButtonTitle || 'Submit';
```

Comment:
If we are passing in the action, I don't really see why this needs to be passed in via options. Just set it in the conditional below.